From 0cdfac2a2dc46c038660272e1a4bf9873c55607c Mon Sep 17 00:00:00 2001
From: Alessandro Astone <ales.astone@gmail.com>
Date: Mar 18 2024 22:35:09 +0000
Subject: Set SDDM cursor size


CursorTheme is passed to kwin_wayland as XCURSOR_THEME, but kwin
ignores it unless XCURSOR_SIZE is also set.

This wasn't necessary in Plasma 5 because sddm-greeter also gets
XCURSOR_THEME and respected it: that's not the case anymore with
Plasma 6.

---

diff --git a/usr/lib/sddm/sddm.conf.d/kde_settings.conf b/usr/lib/sddm/sddm.conf.d/kde_settings.conf
index ce709f7..c025756 100644
--- a/usr/lib/sddm/sddm.conf.d/kde_settings.conf
+++ b/usr/lib/sddm/sddm.conf.d/kde_settings.conf
@@ -2,3 +2,4 @@
 Current=01-breeze-fedora
 # The name comes from /usr/share/icons/breeze_cursors
 CursorTheme=breeze_cursors
+CursorSize=24

